### PR TITLE
Full review of repository code by Maria Bodnar

### PR DESCRIPTION
**Pull request by MaraaaBodnar**

### CODE REVIEW

File: Hrychyna.University.InteractivePlantCareAssistant.Web/script.js [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        return;
+    }
+
+    var threadsList = document.getElementById('threadsList');
+    var li = document.createElement('li');
+
+    var userName = "Dariia";
+    var currentTime = new Date().toLocaleString("uk-UA");
+
+    li.innerHTML = `<span class="thread-title">${threadTitle}</span> <span class="thread-info">(Created by ${userName} at ${currentTime})</span>
```
Comment (MaraaaBodnar) [inline Hrychyna.University.InteractivePlantCareAssistant.Web/script.js:17]: Use template literals for cleaner and more readable HTML string concatenation.

File: Hrychyna.University.InteractivePlantCareAssistant.Web/script.js [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+        <span class="likes" onclick="toggleLike(this)">
+            <svg class="heart" viewBox="0 0 32 29.6">
+                <path d="M23.6,0A8.6,8.6,0,0,1,32,8.6c0,9.1-14.4,21-16,21S0,17.7,0,8.6A8.6,8.6,0,0,1,8.6,0a9.3,9.3,0,0,1,7.4,3.6A9.3,9.3,0,0,1,23.6,0Z"/>
+            </svg>
+            <span class="like-count">0</span>
+        </span>`;
+
+    li.onclick = function(event) {
+        if (event.target.closest('.likes')) return;
```
Comment (MaraaaBodnar) [inline Hrychyna.University.InteractivePlantCareAssistant.Web/script.js:26]: Check if event.target.closest('.likes') is truthy in an if statement directly.

File: Hrychyna.University.InteractivePlantCareAssistant.Web/script.js [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+        // If already liked, decrease the like count and change color to default
+        likes--;
+        element.classList.remove('liked');
+    } else {
+        // If not liked, increase the like count and change color to red
+        likes++;
+        element.classList.add('liked');
+    }
+
+    likeCountSpan.textContent = likes; // Update the display of likes
```
Comment (MaraaaBodnar) [inline Hrychyna.University.InteractivePlantCareAssistant.Web/script.js:57]: Update the display of likes

File: Hrychyna.University.InteractivePlantCareAssistant.Web/script.js [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+
+
+
+
+function goBack() {
+    document.getElementById('threadsView').style.display = 'block';
+    document.getElementById('messagesView').style.display = 'none';
+}
+
+document.getElementById('newThreadTitle').addEventListener('keypress', function(event) {
```
Comment (MaraaaBodnar) [inline Hrychyna.University.InteractivePlantCareAssistant.Web/script.js:123]: Add 'input' event listener as well to handle pasting and other input methods.

File: Hrychyna.University.InteractivePlantCareAssistant.Web/script.js (hunk 1 above)
Comment (DariaHrychyna) [reply Hrychyna.University.InteractivePlantCareAssistant.Web/script.js:17]: Thanks for the comment. I'll leave things as they are for now, but I'll consider this suggestion later.

File: Hrychyna.University.InteractivePlantCareAssistant.Web/script.js (hunk 2 above)
Comment (DariaHrychyna) [reply Hrychyna.University.InteractivePlantCareAssistant.Web/script.js:26]: Thanks for the heads up, I'll look into that suggestion.

**Pull request merged**